analyzers/tests/SonarAnalyzer.UnitTest/TestCases/InfiniteRecursion.RoslynCfg.cs

```diff
@@ -497,6 +497,50 @@ public interface IWithDefaultImplementation
             return Count * Price + Total();
         }
     }
+
+    abstract class One
+    {
+        protected One Parent;
+
+        public virtual int Prop
+        {
+            get
+            {
+                return Parent.Prop;
+            }
+        }
+
+        public virtual int Method()
+        {
+            return Parent.Method();
+        }
```

Comment:
Let's also add a case like this, because it produces the FlowReference stuff
```
        public virtual int? NullableMethod()
        {
            return Parent?.NullableMethod();
        }
```